.circleci/config.yml

```diff
@@ -38,12 +38,6 @@ jobs:
       USE_SYMENGINE: 1
     docker:
       - image: circleci/python:3.7
-  "python-3.7-sympy-1.5":
```

Comment:
We should leave behind a 1.3 test until after the 0.4.5 release

Reply:
Will do